### ISSUE TITLE
[Machine Learning] Explainable AI using ELI5

### ISSUE THREAD

**Issue by jamesomina99** [section-engineering-education/engineering-education#4324]
#### Proposed title of article
[Machine Learning] Explainable AI using ELI5 

#### Proposed article introduction
Explainable AI  is artificial intelligence with an understandable decision-making process. That is, a human can understand and explain how an AI program reached the output it provides. Explainable AI tries to answer the question why should I trust the model?. You should be able to explain what your model is doing. It is not enough to just trust your model prediction accuracy or score.

Explainable AI is an answer to the AI black box problem. When it comes to an AI-powered decision or answer, we tend to only see the start (input) and the result (output). The process in the middle isn’t visible to us. This is the AI black box.
The black box problem is most rampant in machine learning technology and neural networks. Explainable AI tries to explain the hidden functionalities of the model and explains how a model reached a certain conclusion.

With artificial intelligence growing in use, the need for explainable AI becomes more pressing. We’re reaching a point where a flawed AI-based decision can have a heavy impact on someone’s life. For instance, consider medical diagnosis AI, where a wrong answer could result in needless suffering. If we can’t understand why AI gives a certain diagnosis, how can we trust it?

Explainable AI is, in its simplest terms, AI with none of its hidden workings hidden away. It keeps clear the reasoning behind its answers.  So, with it, we can make sure that the decisions made by machines are the best ones. 

In this tutorial, we shall use ELI5 in building our model.  ELI5 is an acronym for ‘Explain Like I’m 5’. It’s a Python library that’s popular because it’s easy to use.  ELI5 is used inspect ML classifiers and explain their predictions. It is popularly used to debug algorithms such as sklearn regressors and classifiers. It’s mainly used:

- To learn about important features that played a crucial role in predictions.
- To analyze a certain individual prediction, and see what exactly led the model to that prediction. 

#### Key takeaways 
- What is Explainable AI.
- Various techniques used for Explainable AI.
- What is ELI5.
- How to install ELI5.
- Building a custom model(Following all the machine learning stages).
- Explaining the custom model using Explainable AI.

#### Article quality
In this tutorial, we will build a custom model from scratch.  We will follow all the stages of building a machine learning model from beginning to end. This tutorial can be followed by anyone since we start from the basics and explain each concept well.

After building the model, we then use this ELI5 to explain the predictions made by the model. This enables us to trust the model so that we can not just the model results blindly.

This is a practical guide and anyone should comfortably follow and learn about Explainable AI.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)
